consensus/src/miner.rs

```diff
@@ -144,15 +144,15 @@ impl MineContext {
         let (nonce, proof) = match mined {
             Err(PoswError::SnarkError(SNARKError::Terminated)) => {
                 // technically a race condition, but non-critical
-                trace!("terminated miner due to canon block received");
+                debug!("terminated miner due to canon block received");
```

Comment:
Perhaps? 
```suggestion
                debug!("terminated miner due to competing canon block received");
```

Reply:
once the new block is recognized as canon, I don't know if it can have any competition per se - an alternative fork might compete with the current canon one, but I don't think it applies to a block of the same height